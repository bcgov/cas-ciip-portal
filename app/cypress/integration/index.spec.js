describe('The index page', () => {
  it('matches the snapshot when there is no session', () => {
    cy.visit('/');
    cy.get('.content');
    cy.document().toMatchSnapshot();
  });
});
