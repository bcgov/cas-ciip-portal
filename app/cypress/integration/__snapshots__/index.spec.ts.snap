exports[`The index page > matches the snapshot when there is no session #0`] = `
<div class="content narrow container">
  <div class="row" style="margin-top: 60px;">
    <div class="col-md-6">
      <h3 class="jsx-473442302 blue">
        What is the CleanBC Industrial Incentive Program?
      </h3>
      <p class="jsx-473442302">
        In 2018, B.C.’s $30 carbon tax rate was raised to $35, and it is set to
        increase by $5 every year until 2021. As the price of carbon rises, the
        CleanBC Program for Industry will support competitiveness and facilitate
        emission reductions using revenues from the carbon tax that industry
        pays above $30 per tonne carbon dioxide equivalent (tCO2e).
      </p>
      <p class="jsx-473442302">
        The CleanBC Industrial Incentive Program (CIIP) is part of the CleanBC
        Program for Industry, which applies to large industrial operations that
        report their emissions under the Greenhouse Gas Industrial Reporting and
        Control Act (GGIRCA).
      </p>
      <p class="jsx-473442302">
        The CIIP helps cleaner industrial operations across the province by
        reducing carbon tax costs for facilities near world-leading emissions
        benchmarks.
      </p>
    </div>
    <div class="col-md-5 offset-md-1">
      <div class="card" style="width: 100%; margin: 30px 0px;">
        <div class="card-body">
          <div class="blue card-title h5">
            Apply for CleanBC Industrial Incentive Program (CIIP)
          </div>
          <p class="card-text" style="padding: 10px 0px;">
            Operators must submit a CIIP application form by June 30, 2019. As
            part of the application, information about the operation’s energy
            use, emissions, and production is required.
          </p>
          <form action="/login" method="post" class="">
            <button
              type="submit"
              class="full-width btn btn-primary btn-lg"
              style="padding: 15px;"
            >
              Register and Apply
            </button>
          </form>
        </div>
      </div>
      <form action="/login" method="post" class="">
        <button type="submit" class="login-link btn btn-outline-dark">
          Already have an account? Click here to login.
        </button>
      </form>
    </div>
  </div>
  <div id="value-props" class="row" style="margin-top: 100px;">
    <div class="col-md-12">
      <h4 class="jsx-473442302 blue">How to Apply</h4>
    </div>
    <div class="col-md-4">
      <div class="jsx-473442302 value-prop">
        <img src="../../static/icons/import.png" class="jsx-473442302" />
        <h5 class="jsx-473442302 blue">
          1. Register as an Industrial Reporter
        </h5>
        <p class="jsx-473442302">
          Before you can apply for an operation, you have to register yourself
          as an industrial reporter.
        </p>
      </div>
    </div>
    <div class="col-md-4">
      <div class="jsx-473442302 value-prop">
        <img src="../../static/icons/production.png" class="jsx-473442302" />
        <h5 class="jsx-473442302 blue">2. Request to apply for an Operation</h5>
        <p class="jsx-473442302">
          Once you’ve registered you can request to apply on behalf one or
          multiple Operations.
        </p>
      </div>
    </div>
    <div class="col-md-4">
      <div class="jsx-473442302 value-prop">
        <img src="../../static/icons/paid.png" class="jsx-473442302" />
        <h5 class="jsx-473442302 blue">3. Apply on behalf of the Operation</h5>
        <p class="jsx-473442302">
          The CIIP team will verify and approve your request and then you can
          apply for the program.
        </p>
      </div>
    </div>
  </div>
  <div class="row" style="margin-top: 100px;">
    <div class="col-md-6">
      <img
        src="../../static/what-is-it.jpg"
        class="jsx-473442302 with-shadow"
      />
    </div>
    <div class="col-md-5 offset-md-1">
      <h3 class="jsx-473442302 blue">Key Dates</h3>
      <table class="table table-striped table-bordered table-hover">
        <thead class="jsx-473442302">
          <tr class="jsx-473442302">
            <th class="jsx-473442302">Date</th>
            <th class="jsx-473442302"></th>
          </tr>
        </thead>
        <tbody class="jsx-473442302">
          <tr class="jsx-473442302">
            <td class="jsx-473442302">Apr 1, 2019</td>
            <td class="jsx-473442302">CIIP application forms open</td>
          </tr>
          <tr class="jsx-473442302">
            <td class="jsx-473442302">May 16, 2020</td>
            <td class="jsx-473442302">Webinar for CIIP</td>
          </tr>
          <tr class="jsx-473442302">
            <td class="jsx-473442302">Dec 30, 2019</td>
            <td class="jsx-473442302">CIIP application form due</td>
          </tr>
          <tr class="jsx-473442302">
            <td class="jsx-473442302">May 31, 2021</td>
            <td class="jsx-473442302">
              CIIP application form due <br class="jsx-473442302" />(along with
              2019 GHG reporting data)
            </td>
          </tr>
        </tbody>
      </table>
      <div
        class="ciip-card card"
        style="width: 100%; margin: 40px 0px; border: 1px solid grey;"
      >
        <div class="card-body">
          <div class="blue card-title h5">Contact Information</div>
          <p class="card-text" style="padding: 10px 0px;">
            Please email us at
            <strong class="jsx-473442302">GHGRegulator@gov.bc.ca</strong> for
            any questions.
          </p>
        </div>
      </div>
      <p class="jsx-473442302">
        Further information on the CleanBC Industrial Incentive Program is
        available in this<a
          target="_blank"
          rel="noopener noreferrer"
          href="https://www2.gov.bc.ca/assets/gov/environment/climate-change/ind/cleanbc-program-for-industry/ciip_factsheet_190524_final.pdf?forcedownload=true"
          class="jsx-473442302"
        >
          fact sheet</a
        >.
      </p>
    </div>
  </div>
  <div class="row" style="margin-top: 100px; display: none;">
    <div class="jumbotron" style="width: 100%;">
      <h3 class="jsx-473442302 blue">Have questions? Get in touch.</h3>
      <p class="jsx-473442302">Phone: +1 099 9920 9002</p>
      <p class="jsx-473442302">Email: cas@ggircs.com</p>
    </div>
  </div>
</div>
`;
